src/server.c

```diff
@@ -52,12 +52,14 @@ int send_response(int fd, char *header, char *content_type, void *body, int cont
 {
     const int max_response_size = 65536;
     char response[max_response_size];
-
+    int response_length = sprintf(response, " Header: %s\n Connection: close\n Content_length: %d\n Content_type: %s\n\n %d\n", header, content_length, content_type, body);
```

Comment:
```suggestion
    int response_length = sprintf(response, "%s\n Connection: close\n Content_length: %d\n Content_type: %s\n\n %d\n", header, content_length, content_type, body);
```
Drop `Header:` from your first `%s`. also, you're calling `body` a `%d`. this will work for the `d20` because you are passing an int.  You will run into issues when you try to pass anything else to this function.  I would call body a`%s` here.